src/main/java/net/coderbot/iris/gl/image/ImageBinding.java

```diff
@@ -23,10 +23,6 @@ public ImageBinding(int imageUnit, int internalFormat, IntSupplier textureID) {
 	}
 
 	public void update() {
-		if (useExt) {
-			EXTShaderImageLoadStore.glBindImageTextureEXT(imageUnit, textureID.getAsInt(), 0, false, 0, GL42C.GL_READ_WRITE, internalFormat);
-		} else {
-			GL42C.glBindImageTexture(imageUnit, textureID.getAsInt(), 0, false, 0, GL42C.GL_READ_WRITE, internalFormat);
-		}
+		IrisRenderSystem.bindImageTexture(useExt, imageUnit, textureID.getAsInt(), 0, false, 0, GL42C.GL_READ_WRITE, internalFormat);
```

Comment:
It might make sense to move the useExt code to IrisRenderSystem too, that counts as a direct GL reference.